web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx

```diff
@@ -171,6 +268,115 @@ export const MiniAppForm = (props: LinksFormProps) => {
           </p>
         )}
       </div>
+
+      <div className="grid gap-y-2">
+        <Typography variant={TYPOGRAPHY.H7}>Supported Countries</Typography>
+
+        <Typography variant={TYPOGRAPHY.R3} className="text-grey-500">
+          If your app is Mini App then you must select a list of countries where
+          Mini App will be available.
+        </Typography>
+
+        <Controller
+          control={control}
+          name="supported_countries"
+          render={({ field }) => (
+            <SelectMultiple
+              values={selectedCountries}
+              onChange={(value) => {
+                if (!field.value) {
+                  return field.onChange([]);
+                }
+
+                field.onChange(
+                  field.value.some((v) => v === value)
+                    ? field.value.filter((v) => v !== value)
+                    : [...field.value, value],
+                );
+              }}
+              onRemove={(value) =>
+                field.onChange(field.value?.filter((v) => v !== value) ?? [])
+              }
+              items={countries}
+              label="Supported Countries"
+              disabled={!isEditable || !isEnoughPermissions}
+              errors={errors.supported_countries}
+              required={appMode}
+              selectAll={() =>
+                field.onChange(
+                  selectedCountries?.length && selectedCountries.length > 0
+                    ? []
+                    : countries.map((c) => c.value),
+                )
+              }
+            />
+          )}
+        />
+      </div>
+
+      <div className="grid gap-y-2">
+        <Typography variant={TYPOGRAPHY.H7}>Supported Languages</Typography>
+
+        <Typography variant={TYPOGRAPHY.R3} className="text-grey-500">
+          If your app is Mini App then you must select a list of languages which
```

Comment:
@andy-t-wang Please check this description. Let me know if we need to update it.